src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -207,13 +207,7 @@ protected virtual JObject GetUnclaimedGas(JArray _params)
         static string GetExceptionMessage(Exception exception)
         {
             if (exception == null) return null;
-
-            if (exception.InnerException != null)
-            {
-                return exception.InnerException.Message;
-            }
-
-            return exception.Message;
+            return exception.GetBaseException().Message;
```

Comment:
It's recursive?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.exception.getbaseexception?view=net-5.0

It's used to get the root cause of the exception, which may  tell the more detailed information.